ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -323,6 +323,11 @@ final class SandboxServer(
           authService.getClass.getSimpleName,
           config.seeding.fold("no")(_.toString.toLowerCase),
         )
+        if (config.scenario.nonEmpty) {
+          logger.withoutContext.warn(
+            """Initializing a ledger with scenarios is deprecated and will be removed in the future. You are advised to use DAML Script instead.
+              |A migration guide for converting your scenarios to DAML Script is available at https://docs.daml.com/daml-script/index.html#using-daml-script-for-ledger-initialization""".stripMargin)
```

Comment:
You can remove the "index.html".

```suggestion
              |A migration guide for converting your scenarios to DAML Script is available at https://docs.daml.com/daml-script/#using-daml-script-for-ledger-initialization""".stripMargin)
```

Reply:
Thanks. Fixed in 6cb255c.